todo.js

```diff
@@ -46,29 +38,27 @@ function addTodo(todo, update) {
   let todoText = document.querySelector('input[name="todo"]').value;
   let priority = document.getElementById("priority").value;
   let id = state.maxId;
-  let endDay = undefined
+  let endDay = new Date().toLocaleDateString();
+  let todoList = [];
 
   if (todo) {
     todoText = todo.todoText;
     id = todo.id;
     priority = todo.priority;
     endDay = todo.endDay;
-    // todoTimeEachDay = todo.todoTimeEachDay; //TODO choose the right todoTimeEachDay
+    todoList = currentTomatoes(id);
```

Comment:
I don't understand what's happening in this code block, could you add an explanatory comment, what is this doing and why?

Reply:
I delete it already, it was some old stuff :)